docs/antora.yml

```diff
@@ -0,0 +1,32 @@
+name: migrations 
+version: ~
+title: Neo4j Migrations Docs
+start_page: ROOT:index.adoc
+nav:
+  - modules/ROOT/nav.adoc
+
+asciidoc:
+  attributes:
```

Comment:
Please note that you cannot (yet) use attribute references. The following *won't* work:

```yaml
    url-gh: https://github.com/michael-simons/neo4j-migrations
    url-gh-releases: {url-gh}/releases
```